lms/static/scripts/frontend_apps/components/dashboard/test/AllCoursesActivity-test.js

```diff
@@ -86,6 +86,16 @@ describe('AllCoursesActivity', () => {
     return wrapper;
   }
 
+  /**
+   *
+   * @param {'students' | 'assignments' | 'courses'} prop
+   */
+  function updateFilter(wrapper, prop, arg) {
```

Comment:
For this internal helper I think the code is straightforward enough that the comment could be omitted. What might help readability is to give the parameters more specific names, eg. `prop` -> `filterProp`, `arg` -> `ids`.

Reply:
I think I'll keep the comment, mostly because it provides IDE autocompletion. I'll rename the params though.